site/src/pages/components/switch.explainer.mdx

```diff
@@ -0,0 +1,574 @@
+  specify a value to align with the [checkbox default value
+  behavior](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value:~:text=If%20the%20value%20attribute%20was%20omitted%2C%20the%20default%20value%20for%20the%20checkbox%20is%20on%2C%20so%20the%20submitted%20data%20in%20that%20case%20would%20be%20subscribe%3Don.)?
+</div>
+
+<div class="question">
+  Should a custom "activate initially" attribute be used with the switch (`toggled`) or should the checkbox `checked` attribute be used as a standard?
+  `<switch toggled>` seems more natural while `<switch checked>` is already an established standard
+</div>
+
+| Property Name | Attribute Name | Type      | Default Value | Description                                                                                                                                                                                                                    |
+| ------------- | -------------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `toggled`(?)  | `toggled`      | `boolean` | `false`       | Indicates if the switch should be "on" initially (similar to the `checked` attribute on the checkbox).                                                                                                                         |
+| `value`       | `value`        | `string`  | `"off"` (?)   | The current value of the slider. Only set if the switch is toggled/on on form submission.                                                                                                                                      |
+| `name`        | `name`         | `string`  | `""`          | The name of the component.                                                                                                                                                                                                     |
+| `id`          | `id`           | `string`  | `""`          | The unique identifier of the component.                                                                                                                                                                                        |
+| `type`        | `type`         | `string`  | `button`      | Similar to the `<button>` or `<input>`, the type property for the switch can take either "button" or "submit". "Button" is used for triggering JavaScript actions, "submit" is used to immediately submit the associated form. |
+| `readonly`    | `readonly`     | `boolean` | `false`       | Indicates that the switch is not interactive but its value should still be submitted with the form.                                                                                                                            |
```

Comment:
in html, readonly isn't allowed on checkboxes - so it'd be odd to have that here.

This table is missing a reference to `disabled` which _should_ be listed here and is allowed on checkboxes / other form elements that don't support readonly

Reply:
I removed readonly and added disabled.